### ISSUE TITLE
Unable to build app due to issue with name of PrivacyInfo file

### ISSUE THREAD

**Issue by taschik** [react-native-google-cast/react-native-google-cast#535]
**Describe the bug**
The version 4.8.1 of the SDK cannot be included in a project running iOS 17.5 building with XCode15.4 due to the naming of the `PrivacyInfo.xcprivacy`. 

Apple requires the filename being reserved for their own PrivacyInfo file and there cannot be multiple files with the same name in the project.

Please rename it to something like `GCPrivacyInfo.xcprivacy` which should resolve the problem.

```
Multiple commands produce '/Users/user/Library/Developer/Xcode/DerivedData/app-fifryutghudbkfccqcndjphjdlvb/Build/Products/Debug-iphonesimulator/myapp.app/PrivacyInfo.xcprivacy'
```

**Expected behavior**
Xcode is able to build the app with the SDK included

**Comment by petrbela** [comment]
I'm not sure what you're doing wrong but 
1. I'm able to build without an issue
2. the `PrivacyInfo.xcprivacy` file is added by `google-cast-sdk`, not this React Native library, so if it's a valid issue, please log it on Google's issue tracker. https://developers.google.com/cast/support

**Comment by petrbela** [comment]
Apparently, it should be fixed in 4.8.1.1, make sure to update your pods https://issuetracker.google.com/issues/336329420
